test/org/sagebionetworks/bridge/models/itp/IntentToParticipateTest.java

```diff
@@ -26,16 +26,17 @@ public void canSerialize() throws Exception {
                 .withSignedOn(TIMESTAMP).withConsentCreatedOn(TIMESTAMP).build();
         
         IntentToParticipate itp = new IntentToParticipate.Builder().withStudyId("studyId").withPhone(PHONE)
-                .withSubpopGuid("subpopGuid").withScope(SharingScope.ALL_QUALIFIED_RESEARCHERS).withOsName("iOS")
-                .withConsentSignature(consentSignature).build();
+                .withEmail(EMAIL).withSubpopGuid("subpopGuid").withScope(SharingScope.ALL_QUALIFIED_RESEARCHERS)
+                .withOsName("iOS").withConsentSignature(consentSignature).build();
```

Comment:
Should we also check that deserializing puts the Email back in the Java Object?

Reply:
Added